packages/protocol/test/identity/escrow.ts

```diff
@@ -72,10 +72,15 @@ contract('Escrow', (accounts: string[]) => {
   const owner = accounts[0]
   let registry: RegistryInstance
 
+  before(async () => {
+    registry = await getDeployedProxiedContract('Registry', artifacts)
+    // Take ownership of the registry contract to point it to the mocks
+    await assumeOwnership(['Registry'], owner)
+  })
+
   beforeEach(async () => {
-    registry = await Registry.new(true)
     escrow = await Escrow.new(true, { from: owner })
-    await escrow.initialize(registry.address)
+    await escrow.initialize()
```

Comment:
The initialize() function no longer takes in an argument, does it?

Reply:
exactly! no longer needs to set the registry address at all, as this is now the constant val